clang/test/CodeGenSYCL/mark-kernel-name.cpp

```diff
@@ -1,14 +1,19 @@
-// RUN: %clang_cc1 -triple x86_64-linux-pc  -fsycl-is-host -disable-llvm-passes -emit-llvm %s -o - | FileCheck %s
-// RUN: %clang_cc1 -triple x86_64-linux-pc  -fsycl-is-device -disable-llvm-passes -emit-llvm %s -o - | FileCheck %s
+// RUN: %clang_cc1 -triple x86_64-linux-pc -fsycl-is-host -disable-llvm-passes -fdeclare-spirv-builtins -emit-llvm %s -o - | FileCheck %s
+// RUN: %clang_cc1 -triple spir64-sycldevice -aux-triple x86_64-linux-pc  -fsycl-is-device -disable-llvm-passes -emit-llvm %s -o - | FileCheck %s
```

Comment:
Do we need the aux-triple here?

Reply:
yeah, it makes the spir builtins from the mock header work.